livechat/overview/get_started.rst

```diff
@@ -0,0 +1,83 @@
+==========================
+Get Started with Live Chat
+==========================
+
+Real-time chat is a cost-effective tool and a great way to bring visitors back to your website. It
+is an accessible and convenient customer support channel, and can increase efficiency, considering
+that this way, support representatives can handle more than one case at a time.
```

Comment:
```suggestion
is an accessible and convenient customer support channel, and can increase efficiency. Considering
that this way, support representatives can handle more than one case at a time.
```

Reply:
I think that if it is for the line to be broken into two-paragraphs I'd have to rework on everything after the first colon. So, as I believe the sentences nicely flow between each other, and the whole paragraph is easy to read and understand the way I wrote it, I won't accept the suggestion (: